tools/state-viewer/src/latest_witnesses.rs

```diff
@@ -1,12 +1,36 @@
+use std::path::{Path, PathBuf};
 use std::rc::Rc;
 
-use clap::Parser;
-use near_chain::ChainStore;
+use near_async::time::Clock;
+use near_chain::runtime::NightshadeRuntime;
+use near_chain::stateless_validation::processing_tracker::ProcessingDoneTracker;
+use near_chain::{Chain, ChainGenesis, ChainStore, DoomslugThresholdMode};
+use near_epoch_manager::shard_tracker::{ShardTracker, TrackedConfig};
+use near_epoch_manager::EpochManager;
+use near_primitives::stateless_validation::ChunkStateWitness;
 use near_primitives::types::EpochId;
 use near_store::Store;
 use nearcore::NearConfig;
+use nearcore::NightshadeRuntimeExt;
 
-#[derive(Parser)]
+#[derive(clap::Subcommand)]
+pub enum StateWitnessCmd {
```

Comment:
Can you add a short comment on how to use both commands to first dump the witness to a file and then validate it? Also it would be nice to make it so that it's visible in the --help of this subcommand.

Reply:
Help for subcommands is taken from comment for the parent command, so I put the comment into `StateViewerSubCommand::StateWitness`.